ansible/playbook.yml

```diff
@@ -9,11 +9,13 @@
     - role: svc_traefik
     - role: svc_minio
     - role: svc_coredns
+    - role: svc_prometheus
+    - role: svc_node_exporter
     - role: deerhide_users
 
-- name: Rasp's playbook
-  hosts:
-    - rasp
-  roles:
-    - role: svc_docker
-    - role: deerhide_users
+# - name: Rasp's playbook
+#   hosts:
+#     - rasp
+#   roles:
+#     - role: svc_docker
+#     - role: deerhide_users
```

Comment:
Is this intended? If not, please reenable this playbook before it goes to main.

Reply:
Done